app/controlplane/internal/conf/conf.proto

```diff
@@ -59,10 +59,17 @@ message Server {
     google.protobuf.Duration timeout = 5;
   }
 
+  message TLS {
```

Comment:
It would be great if you could also add an example in the same files (both CAS and CP) https://github.com/chainloop-dev/chainloop/blob/main/app/controlplane/configs/samples/config.yaml